lz4/PKGBUILD

```diff
@@ -0,0 +1,127 @@
+# Maintainer: Sébastien Luttringer
+# PGO build: Laio O. Seman <laio [at] ieee.org>
+
+pkgname=lz4
+epoch=1
+pkgver=1.9.4
+pkgrel=2
+pkgdesc='Extremely fast compression algorithm'
+arch=('x86_64')
+url='https://lz4.github.io/lz4/'
+license=('GPL2')
+makedepends=('git' 'cmake' 'ninja', 'llvm' 'clang')
+checkdepends=('diffutils')
+depends=('glibc')
+source=("git+https://github.com/lz4/lz4.git#tag=v$pkgver")
+sha256sums=('SKIP')
+
+# Function to generate data and compress-decompress it
+local generate_and_process_data() {
+    echo "Generating data for $1 KB with compression level $2"
+    DATA_SIZE=$1
```

Comment:
```suggestion
    local data_size=$1
```

Reply:
all variables are now private inside the functions, unless they are not (accessing global ones)

Reply:
> 

specifying an underscore before a function name is just a convention for naming functions called only inside a script. Variables inside a function do not need to have an underscore in front of them, they should be declared as local variables to avoid clogging up the global environment.